examples/kfpytorch_plugin/kfpytorch_plugin/pytorch_mnist.py

```diff
@@ -350,6 +350,14 @@ def pytorch_training_wf(
 # To visualize the outcomes, you can point Tensorboard on your local machine to these storage locations.
 # :::
 #
+# :::{note}
```

Comment:
Bit of scope creep:
I'm moving this section up before the "pytorch elastic" section as this affects only `task_config=Pytorch` tasks. Tasks using `task_config=Elastic` do this by default [here](https://github.com/flyteorg/flytekit/blob/a2bcf95048962b77ba2e67052b8f4a8c956fe81f/plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py#L497).